src/app/(articles)/articles/[slug]/page.tsx

```diff
@@ -92,34 +88,18 @@ const Page = async ({ params }: { params: { slug: string } }) => {
       />
 
       {/* Hero Image Section - Static Images */}
-      {hasStaticMainImage && article?.image && (
-        <>
-          <figure className="relative h-[350px] w-full overflow-hidden rounded-10 sm:h-[400px]">
-            <Image
-              src={article?.image}
-              alt={`${article.title} article image`}
-              fill
-              className=" mx-auto h-full w-full object-cover object-center"
-              style={{ "--index": 2 } as React.CSSProperties}
-              priority
-              quality={100}
-            />
-          </figure>
-          <div className="h-16" />
-        </>
-      )}
-      {/* Hero Image Section - Dynamic Images */}
-      {hasDynamicMainImage && article?.dynamicMainImage && (
+      {article?.image && (
         <>
           <figure className="relative h-[350px] w-full overflow-hidden rounded-10 sm:h-[400px]">
             <Image
-              src={article?.dynamicMainImage}
+              src={`${article?.image}`}
               alt={`${article.title} article image`}
               fill
               className=" mx-auto h-full w-full object-cover object-center"
               style={{ "--index": 2 } as React.CSSProperties}
               priority
               quality={100}
+              sizes="(min-width: 1024px) 32rem, 20rem"
```

Comment:
For the main image, the default width will always be 3840 when you open the image in a new tab. This will enlarge the image size. This should be set to a lower number (e.g. 1080, not sure if this number is the correct approach) to have a smaller image size.